src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -522,7 +528,14 @@ private void ParseNamespaceBody(ref SyntaxToken openBrace, ref NamespaceBodyBuil
                             // incomplete members must be processed before we add any nodes to the body:
                             AddIncompleteMembers(ref pendingIncompleteMembers, ref body);
 
-                            body.Members.Add(this.ParseNamespaceDeclaration());
+                            var attributeLists = _pool.Allocate<AttributeListSyntax>();
+                            var modifiers = _pool.Allocate();
```

Comment:
>var [](start = 28, length = 3)

nit: type should be spelled out since not visible on the RHS

Reply:
I just checked, and every other occurrence of this in the rest of hte file uses 'var'.  :-/    I would prefer teh compiler side just be consistent.  If you guys *do* want to have rules about var, could you at least go sweep the code to be consistent ith that so that then new PRs don't have to decide about if they should follow those rules, or be consistent with the existing code.  It's esp annoying when you move code around.  Should you fixup then?  or just move and make the change as minimal as possible.